File: spojPractice/spoj42.cpp
#include <iostream>
#include <string> //for strings
#include <algorithm> //for reversing strings
#include <sstream> //for converting 
using namespace std;
int reverse(string s)
{
	int x;
	reverse(s.begin(),s.end()); //reverses the string
	stringstream convert(s); //convert is object of stringstream
	convert>>x; //code used to convert string to int
	return x;
}
int main()
{
	int t;
	cin>>t;
	for(int q=1;q<=t;q++)
	{
		string a,b;
		cin>>a>>b;
		int c=reverse(a);
		int d=reverse(b);
		//cout<<"revers a "<<c<<endl;
		//cout<<"revers b "<<d<<endl;
		int sum=c+d;
		stringstream convert; //convert is object of stringstream
		convert<<sum; //converting int to string 
		int ans=reverse(convert.str()/*convert.str()outputs the sum int as sum string*/);
		cout<<ans<<endl;
	}
}

File: spojPractice/spoj42.py
def reverse(n):
	return n/10
t=int(input())
while (t)!=0:
	a,b=map(str,input().split())
	#a=input()
	#b=input()
	a=a[::-1]
	b=b[::-1]
	c=int(a)+int(b)
	c=str(c)
	c=c[::-1]
	print (int(c))
	t=t-1